instructions.md

```
@@ -0,0 +1,49 @@
+## 1. Database Setup
+The Prisma schema has been updated to include the `AnalyticReport` model. You need to push these changes to your database.
+
+Run the following command:
+```bash
+npx prisma db push
+```
+
+## 2. Firebase Storage
+Ensure your Firebase Storage rules allow read and write operations. For development, you can use:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)